clarifai/runners/models/model_run_locally.py

```diff
@@ -475,7 +475,7 @@ def main(model_path,
          keep_env=False,
          keep_image=False):
 
-  if not os.environ['CLARIFAI_PAT']:
+  if 'CLARIFAI_PAT' not in os.environ:
```

Comment:
might want 
```
if not os.environ.get("CLARIFAI_PAT", None)
```
so that it validates empty PATs as incorrect as well

Reply:
yeah I guess that is also a possibility.